### PR TITLE
fix(controller): storage file service support put method

### PR DESCRIPTION
**Pull request by anda-ren**
## Description

1. make file type object store support file type
2. server signed url are sourced from request server name
cc @tianweidut 

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/fs/StorageAccessServiceFile.java [hunk 1]
```diff
@@ -167,11 +171,28 @@ public void delete(String path) throws IOException {
...
     @Override
     public String signedUrl(String path, Long expTimeMillis) throws IOException {
-        return serviceProvider + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
+        return possibleServerUrl() + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
     }
 
     @Override
     public String signedPutUrl(String path, String contentType, Long expTimeMillis) throws IOException {
-        return serviceProvider + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
+        return possibleServerUrl() + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
```
[the remark this answers is not in the document]
Comment (anda-ren) [reply server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/fs/StorageAccessServiceFile.java]: signature is added to the signed url

File: server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/fs/StorageAccessServiceFile.java [hunk 2]
```diff
@@ -167,11 +174,30 @@ public void delete(String path) throws IOException {
...
-        return serviceProvider + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
+        String sign = fsStorageSignature.sign(path, expTimeMillis);
+        return possibleServerUrl() + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis) + "?sign=" + sign;
     }
 
     @Override
     public String signedPutUrl(String path, String contentType, Long expTimeMillis) throws IOException {
-        return serviceProvider + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
+        String sign = fsStorageSignature.sign(path, expTimeMillis);
+        return possibleServerUrl() + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis) + "?sign=" + sign;
```
Comment (jialeicui) [inline server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/fs/StorageAccessServiceFile.java:184]: Sign using `(System.currentTimeMillis() + expTimeMillis)` ?

Comment (anda-ren) [reply server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/fs/StorageAccessServiceFile.java]: done

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**